master/internal/provisioner/aws_config.go

```diff
@@ -31,6 +31,19 @@ type AWSClusterConfig struct {
 	LogStream string `json:"log_stream"`
 }
 
+var defaultAWSImageID = map[string]string{
+	"us-west-2": "ami-00f470c0ed034f8e7",
+	"us_east_2": "ami-0b79433826c228d35",
+	"us_east_1": "ami-084469676da11fa5e",
+	"eu_west_2": "ami-07f352b002ba64694",
+	"eu_west_1": "ami-0af0ccbdace44fb31",
+	"eu_central_1": "ami-0da16c489f7e24e17",
+	"ap_southeast_2": "ami-0d00733b08d204db6",
+	"ap_southeast_1": "ami-07b3ac92dcc5d3eb6",
+	"ap_northeast_2": "ami-00aaa2fef0172090b",
+	"ap_northeast_1": "ami-05fad074898b6711f",
```

Comment:
please sort this list (highlight it and run `:sort` in vim) so that the order is identical to all the other places we list all of the regions.

Reply:
Will do